package/com.unity.formats.usd/Runtime/Scripts/IO/Geometry/InstanceImporter.cs

```diff
@@ -68,14 +69,13 @@ public static void BuildSceneInstances(PrimMap primMap, SceneImportOptions optio
             SceneImportOptions options)
         {
             Matrix4x4[] transforms = sample.ComputeInstanceMatrices(scene, pointInstancerPath);
-            int i = 0;
 
             foreach (var protoRoot in sample.prototypes.targetPaths)
```

Comment:
Does "proto" here mean prototype?
could we maybe change this to "prototype" and for below "prototypeRoot" if so?

Reply:
Sure :)